src/KeyFrame.cc

```diff
@@ -777,6 +811,67 @@ void KeyFrame::EraseBackLinksFromPoints()
   
 }
 
+KeyFrame* KeyFrame::CopyKeyFramePartial(MultiKeyFrame* sourceMKF, std::string name)
+{
+	KeyFrame* returnKF = new KeyFrame(sourceMKF, name); //construct a new keyframe
```

Comment:
does name refer to a specific camera image in a multi-keyframe? Can the name "name" be more clear?